### PR TITLE
Adds sea otter above component

### PR DESCRIPTION
**Pull request by daniellemaxwell**
### Scope of changes

Adds sea otter image above the `Set Up A New Project` component and fixes the API Key table so that the line below the header appears.

Also made the following minor adjustments:
- Moved `Status` field to 2nd column of each table (SC-17151)
- `+ Create Project` size is medium to match the `+ New Member` button on the `Teams` page (SC-16734)
- Removed `ID` from the `Organization` table (SC-16319)
- Adds logo for OpenGraph image (SC-16485)
- Fixes issue where email validation error appeared in lowercase and now matches the message that appears in the Login form. (Included in SC-17134)

Note: The font-size for the table headers now appears to be 12px and may not require additional changes.

Fixes SC-17219

### Type of change

- [x] new feature
- [x] bug fix
- [ ] documentation
- [ ] testing
- [ ] technical debt
- [ ] other (describe)

### Acceptance criteria

https://www.awesomescreenshot.com/video/17528143?key=49e9a69d68bedaf3e050dcde4f18134a

https://www.awesomescreenshot.com/image/39965686?key=f425851bfbf2fbc080f443ad70ed2f0b

### Author checklist

- [x] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ]  I have updated the dependencies list
- [ ]  I have recompiled and included new protocol buffers to reflect changes I made
- [ ]  I have added new test fixtures as needed to support added tests
- [x]  Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
- [x]  I have moved the associated Shortcut story to "Ready for Review"

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.
- [ ] Are there any TODOs in this PR that should be turned into stories?

### CODE REVIEW

File: web/beacon-app/src/features/home/components/QuickStart.tsx [hunk 1]
```diff
@@ -1,9 +1,12 @@
+import OtterLookingDown from '@/components/icons/otter-looking-down';
+
 import SetupNewProject from '../components/SetupNewProject';
 import AccessDocumentationStep from './AccessDocumentationStep';
 
 export default function QuickStart() {
   return (
-    <div className="space-y-10">
+    <div className="relative w-full space-y-10">
```
Comment (daniellemaxwell) [inline web/beacon-app/src/features/home/components/QuickStart.tsx:8]: `Relative` and `w-full` were added to make sure the image doesn't move too much when a user adjusts their screen width.

File: web/beacon-app/src/features/projects/components/APIKeysTable.tsx [hunk 2]
```diff
@@ -137,7 +137,6 @@ export const APIKeysTable = ({ projectID }: APIKeysTableProps) => {
       </div>
       <Table
         trClassName="text-sm"
-        className="w-full"
```
Comment (daniellemaxwell) [inline web/beacon-app/src/features/projects/components/APIKeysTable.tsx:140]: This is why the line didn't appear below the header in the API Key table.

File: web/beacon-app/src/features/auth/components/Register/schemas/registrationFormValidation.ts [hunk 3]
```diff
@@ -4,7 +4,7 @@ import { NewUserAccount } from '../../../types/RegisterService';
 
 const validationSchema = Yup.object().shape({
   name: Yup.string().required('The name is required.'),
-  email: Yup.string().email().required('The email address is required.'),
+  email: Yup.string().email('Email is invalid.').required('The email address is required.'),
```
Comment (daniellemaxwell) [inline web/beacon-app/src/features/auth/components/Register/schemas/registrationFormValidation.ts:7]: This message matches the message that appears in the `Login` form when an email without an `@` symbol is typed into the form.

File: web/beacon-app/src/features/home/components/QuickStart.tsx [hunk 4]
```diff
@@ -1,9 +1,12 @@
...
+      <OtterLookingDown className="2xl:right-18 hidden md:absolute md:-top-[215px] md:left-8 md:block md:scale-[.65] xl:left-3/4" />
```
Comment (masskoder) [inline web/beacon-app/src/features/home/components/QuickStart.tsx:9]: I will suggest reviewing the position of the otter to be visible from medium (`md`) to the big screen(`2xl`) and on the same place

File: web/beacon-app/src/features/home/components/QuickStart.tsx (hunk 1 above)
Comment (masskoder) [inline web/beacon-app/src/features/home/components/QuickStart.tsx:8]: Would you mind reviewing the margin of the title  ?

**Review by masskoder — APPROVED** [review]
Looks good just few suggestion to improve the user experience

**Pull request merged**